To: vim_dev@googlegroups.com
Subject: Patch 7.3.127
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.127
Problem:    Compiler complains about comma.
Solution:   Remove comma after last enum element.
Files:	    src/ex_cmds2.c


*** ../vim-7.3.126/src/ex_cmds2.c	2010-12-17 18:06:00.000000000 +0100
--- src/ex_cmds2.c	2010-12-17 19:58:37.000000000 +0100
***************
*** 1115,1121 ****
  static enum
  {
      PEXP_SUBCMD,	/* expand :profile sub-commands */
!     PEXP_FUNC,		/* expand :profile func {funcname} */
  } pexpand_what;
  
  static char *pexpand_cmds[] = {
--- 1115,1121 ----
  static enum
  {
      PEXP_SUBCMD,	/* expand :profile sub-commands */
!     PEXP_FUNC		/* expand :profile func {funcname} */
  } pexpand_what;
  
  static char *pexpand_cmds[] = {
*** ../vim-7.3.126/src/version.c	2011-02-25 14:40:51.000000000 +0100
--- src/version.c	2011-02-25 14:45:19.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     127,
  /**/

-- 
   LAUNCELOT leaps into SHOT with a mighty cry and runs the GUARD through and
   hacks him to the floor.  Blood.  Swashbuckling music (perhaps).
   LAUNCELOT races through into the castle screaming.
SECOND SENTRY: Hey!
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
